core/server/src/main/java/alluxio/master/journal/JournalWriter.java

```diff
@@ -179,16 +213,15 @@ private synchronized OutputStream openCurrentLog() throws IOException {
    */
   private synchronized void deleteCompletedLogs() throws IOException {
     LOG.info("Deleting all completed log files...");
-    // Loop over all complete logs starting from the beginning.
-    // TODO(gpang): should the deletes start from the end?
```

Comment:
Do you think this is still relevant? I think I put in this comment since we do not delete completed logs "atomically". However, with your change, it seems like the deletion of completed logs can look atomic now? if something bad happens during the deletion, it will continue the deletion on the next startup, right?